client/my-sites/stats/stats-subscribers-overview/index.tsx

```diff
@@ -35,9 +35,9 @@ function calculateQueryDate( daysToSubtract: number ) {
 }
 
 // calculate the stats to display in the cards
-function SubscribersOverviewCardStats( subscribersData: SubscribersData[][] ) {
+function SubscribersOverviewCardStats( subscribersDataArrays: subscribersDataArrays[][] ) {
```

Comment:
IMHO L13, L19-20 look okay to me and no need to change.

The only line I'd like changes is just L38, and the name `subscribersDataArrays` sounds good to me.

Let's say we have a type `SubscriberData`, then a good variable name for type `SubscriberData[]` could be `subscriberDataArray` or `subscriberDatas` (latter not so good). And a good variable name for type `SubscriberData[][]` could be just like what you have here `subscribersDataArrays`.

Reply:
Thank you so much for the feedback. Totally get what you're saying here now! Sorry, I totally misunderstood, but when I stop to think about it this makes much more sense. 😄